tests/spec/puppeteer-axe-spec.js

```diff
@@ -0,0 +1,37 @@
+/* Puppeteer tests for the A11y. */
+const { AxePuppeteer } = require('axe-puppeteer');
+const puppeteer = require('puppeteer');
+
+describe("Puppeteer accessibility tests for the Python Editor.", function() {
+    'use strict';
+
+    const editorURL = 'http://localhost:5000/editor.html';
+    let browser = null;
+
+    beforeAll(async () => {
+        // Setup a headless Chromium browser.
+        // Flags allow Puppeteer to run within a container.
+        browser = await puppeteer.launch({
+            headless: true,
+            args: ["--no-sandbox", "--disable-setuid-sandbox", "--disable-dev-shm-usage"]
+        });
+        const page = await browser.newPage();
+ 
+        await page.goto(editorURL);    
+ 
+        await page.close();
+        });
+
+    afterAll(async () => {
+    browser.close();
+    });
+
+    test( 'Checks the editor.html page with Axe', async () => {
+        // First, run some code which loads the content of the page.
+        const page = await browser.newPage();
+        await new AxePuppeteer(page).analyze();
+        await expect(page).toPassAxeTests({
+            disabledRules: [ 'document-title', 'page-has-heading-one', 'html-has-lang', 'landmark-one-main' ],
```

Comment:
`landmark-one-main`:
> Ensures the document has only one main landmark and each iframe in the page has at most one main landmark

Isn't this about having a single "role=main"? We should be passing this test too?

Reply:
The test suggests the test is looking for an element or role, so this should pass, but I can't get it to at present. Unless there are other landmarks that this is checking against, but I can't see anything obvious
 https://dequeuniversity.com/rules/axe/3.4/landmark-one-main?application=axe-puppeteer

Reply:
And it doesn't pass when running against the editor (not the versioner)?

Reply:
No, it doesn't currently.

Reply:
We should check why it's failing then and fix it in our side if it's a problem in the Editor, or raise an issue if it's a bug in the checker.

Reply:
As comment above, was not checking against editorURL